spec/test-data/translations/ios_lint_localizations/extra-strings-in-translations-error.yaml

```diff
@@ -0,0 +1,10 @@
+test_data:
+  en:
+    '"base_string" = "This is in the base language";'
+  fr:
+    '"base_string" = "This is in French";
+     "extra_string" = "This string only exists in French";'
```

Comment:
nitpick: instead of using single quotes around the whole (multiline) string (which I don't think would keep the literal newlines to begin with), what about using `|-` string block syntax?

```suggestion
  en: |-
    "base_string" = "This is in the base language";
  fr: |-
    "base_string" = "This is in French";
    "extra_string" = "This string only exists in French";
```

<img width="1407" alt="image" src="https://github.com/user-attachments/assets/16304a58-b093-4157-9526-6fe3ea671a6b" />